src/A11yDialog.vue

```diff
@@ -73,16 +74,33 @@
       }
     },
 
-    data: () => ({
-      dialog: null
-    }),
+    data () {
+      return {
+        dialog: null
+      }
+    },
 
     methods: {
+      open () {
```

Comment:
How do/will these play with the case when the consumer asks for the `dialog-ref` instance? Wouldn't this neuter or create a double callback situation? I suppose both would be handled by the consumer so I suppose they would be silly to both utilize the instance and also listen for the callback. But maybe there's a case where consumer wants to control opening/closing via the instance, but still listen for these events? Also, can the consumer not just do this themselves via the instance 🤔